src/cow-react/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList.ts

```diff
@@ -15,11 +34,31 @@ export function useLimitOrdersList(): LimitOrdersList {
   const allNonEmptyOrders = useOrders({ chainId })
   const accountLowerCase = account?.toLowerCase()
 
+  const ordersFilter = useCallback((order: Order) => order.owner.toLowerCase() === accountLowerCase, [accountLowerCase])
+  const ordersSorter = (a: Order, b: Order) => Date.parse(b.creationTime) - Date.parse(a.creationTime)
```

Comment:
I don't think we need to create this function every time, right. It's purely static

should this be in a some utils?

Reply:
In my opinion this makes it clean, but no need to move this to utils

Reply:
- Can you elaborate why is cleaner to declare a function inline when it can be a static function? 
- What do you do if you need to sort by creation time somewhere else too? Will you also implement it inline? 
- Unrelated nit: I don't think `a` and `b` are great names for vars.